### PR TITLE
Build docker image on travis

### PR DESCRIPTION
**Pull request by sebeder**
This does that specmate is pushed to the docker hub into the sebeder/specmate repository.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,37 +1,26 @@
 FROM ubuntu:18.04
```
Comment (junkerm) [inline Dockerfile:1]: Is there a reason why you don't build upon the official adoptopenjdk docker image?

File: Dockerfile [hunk 2]
```diff
@@ -1,37 +1,26 @@
...
-RUN npm run build-prod
 
-WORKDIR /opt/specmate/src/bundles
-RUN sed -i 's/\r$//' ./gradlew
-RUN sed -i 's/^M$//' ./gradlew
-RUN ./gradlew --version
-RUN ./gradlew --no-daemon clean build export -x check
-
-RUN cp specmate-std-env/generated/distributions/executable/dev-specmate-all.jar /opt/specmate/specmate.jar
+COPY bundles/specmate-std-env/generated/distributions/executable/dev-specmate-all.jar /opt/specmate/specmate.jar
```
Comment (junkerm) [inline Dockerfile]: would it make semnse to use the prod version?

File: .travis.yml [hunk 3]
```diff
@@ -3,34 +3,35 @@ language:
...
-- cd bundles/specmate-std-env/generated/distributions/executable/
-- mv dev-specmate-all.jar dev-specmate-all-$TRAVIS_TAG.jar 
-- mv prod-specmate-all.jar prod-specmate-all-$TRAVIS_TAG.jar
-- mv prod-specmate-cdo-server.jar prod-specmate-cdo-server-$TRAVIS_TAG.jar
-- mv prod-specmate-no-cdo-server.jar prod-specmate-no-cdo-server-$TRAVIS_TAG.jar
-- cd ../../../../..
+- (cd web && npm run build-prod)
+- (cd bundles && ./gradlew --no-daemon build --continue -x check)
+- (cd bundles && ./gradlew --no-daemon export -x check)
+# - (cd bundles/specmate-std-env/generated/distributions/executable/ && java -Xmx6000m -Djdk.crypto.KeyAgreement.legacyKDF=true -jar dev-specmate-all.jar &)
```
Comment (junkerm) [inline .travis.yml]: remove the uncommented lines, or is there a reason you keep them?

**Review by junkerm — CHANGES_REQUESTED** [review]
looks good, i have some questions and comments

File: Dockerfile (hunk 1 above)
Comment (sebeder) [reply Dockerfile:1]: It was easier to install the other dependencies on ubuntu that on the official adoptopenjdk image.

File: Dockerfile (hunk 2 above)
Comment (sebeder) [reply Dockerfile]: Would it? I don't have stakes in that. This was rather an arbitrary decision.

File: .travis.yml (hunk 3 above)
Comment (sebeder) [reply .travis.yml]: This disables the UI-Tests (because they were unstable). That's why we should keep them.

File: Dockerfile (hunk 2 above)
Comment (junkerm) [reply Dockerfile]: for the docker version I would use the prod version as it removes some unnecssary bundles

File: Dockerfile (hunk 1 above)
Comment (junkerm) [reply Dockerfile:1]: ok!

File: .travis.yml (hunk 3 above)
Comment (junkerm) [reply .travis.yml]: ok!

**Review by junkerm — APPROVED** [review]

**Pull request merged**